src/components/CreateVmWizard/steps/Networking.js

```diff
@@ -337,10 +375,11 @@ class Networking extends React.Component {
     const actionCreate = !!this.state.creating && this.state.creating === rowData.id
     const editedRow = this.state.editing[rowData.id]
 
-    // TODO: Add field level validation for the edit or create fields
-
-    this.props.onUpdate({ [actionCreate ? 'create' : 'update']: editedRow })
-    this.handleRowCancelChange(rowData)
+    const { nicInvalidName, nicDuplicateName } = this.state.editingErrors
```

Comment:
You could do this generically, and just check to make sure every key in `editingErrors` is not true...

```jsx
  const editingErrors = false
  for (const errorKey of this.state.editingErrors) {
    editingErrors = editingErrors || this.state.editingErrors[errorKey]
  }

  if (!editingErrors) {
    ...
  }
```